### PR TITLE
Updating Service Details screenshot

### PR DESCRIPTION
**Pull request by beanh66**
@serenamarie125 I updated the service details page for parent and child services so the tabbed format is gone and the new design is reflected. Let me know if you have feedback. In the upcoming sprint once myHeadings details are worked out, I will create a PR for additional information about this section of services. For now there are placeholder sections listed.

### CODE REVIEW

**Comment by serenamarie125** [comment]
@cshinn Can you do a quick review ?

**Comment by cshinn** [comment]
@beanh66 I know we discussed putting the top row of actions on the left, but I don't remember what we settled on. Just making sure that right aligned is what you intended.

File: UX/ui-service/MyServices/design.md [hunk 1]
```diff
@@ -21,16 +21,18 @@
...
 * The breadcrumbs will change to show My Services as a link back to the List View.
 * The service details will include the following view types: summary view, utilization dashboard, and topology view. The default should be the Summary View.
 
 #### Service - Summary View
 ![Image of Detail View for a Service.](img/Services-DetailView.png)
-* The bottom portion of the Summary View will have a tabbed format with a default on the Resources tab.
-* There will be a tab for Relationships, where the child services should be listed.
+* The summary view will include the properties, resources, and relationships.
 * Child Services are links and will bring the user to the child detail page.
+1. Properties:
```
Comment (cshinn) [inline UX/ui-service/MyServices/design.md]: Was there supposed to be something after these words?

Comment (beanh66) [inline UX/ui-service/MyServices/design.md]: @cshinn Not yet, just a placeholder until we get more information :)

**Comment by beanh66** [comment]
@cshinn Agreed. After spending more time with the design, I think we should talk about the button placement a bit more before we make that decision across the product so for now I left them as right aligned!

**Comment by cshinn** [comment]
@beanh66 makes sense to me

File: UX/ui-service/MyServices/design.md (hunk 1 above)
Comment (beanh66) [inline UX/ui-service/MyServices/design.md]: @cshinn To avoid confusion, I added the basic information I have for these sections.

**Pull request merged**